app-web/src/utils/hooks.js

```diff
@@ -52,15 +52,27 @@ export const useSearch = query => {
   const index = searchClient.initIndex(`Devhub-Algolia-${ALGOLIA_INDEX_SUFFIX}`);
   useEffect(() => {
     if (query) {
-      // if no query, I.E query is '', algolia will return all index....
-      index
-        .search({
-          query: query,
-          hitsPerPage: 200, //set it to a large number, the default value is 200, which some search result is over that number
-        })
-        .then(res => {
-          setResults(res.hits);
-        });
+      if (query.includes('personas:')) {
+        let personaName = query.split(':')[1];
+        const filters = 'persona :' + personaName;
+        index
```

Comment:
those two querieis could be refacgtored. only thing that changes is the search object